bool active
bool bug0
---
string result
